api/routes/analysis.py

```diff
@@ -309,3 +320,67 @@ async def get_sample_reads_map_for_seqr(
         media_type='text/tab-separated-values',
         headers={'Content-Disposition': f'filename={basefn}.tsv'},
     )
+
+
+@router.post('/sample-file-sizes', operation_id='getSampleFileSizes')
```

Comment:
I think the `project_names` will never be large enough to exceed that max query param length, could we make this a get request?

Reply:
Unless there's a way to do this that I don't know, when you have an array input FastApi automatically makes it part of the request_body. It then throws an error that you cannot have a request_body in a get request.

Reply:
Ah figured it out